lib/components/SelectionControl/ToggleSwitch/toggle-switch.scss

```
@@ -0,0 +1,66 @@
+@import "@quill/color.scss";
+@import "@quill/color.opacity.scss";
+
+.toggle-switch {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    user-select: none;
+    width: 48px;
+    height: 28px;
+    border-radius: 18px;
+    border-width: 2px;
+    &.disabled .toggle-switch__slider {
+        pointer-events: none;
+        background: var(--core-color-opacity-black-200);
+    }
+    .toggle-switch__slider {
+        display: flex;
+        align-items: center;
+        justify-content: flex-start;
+        width: 44px;
+        height: 24px;
+        background-color: var(--core-color-opacity-black-200);
+        border-radius: 16px;
+        transition: background-color 0.3s ease;
+        cursor: pointer;
+        position: relative;
+        &:hover:not(.disabled) {
+            background-color: var(--core-color-opacity-black-300);
+        }
+
+        &:active:not(.disabled) {
+            background-color: var(--core-color-opacity-black-400);
+        }
+
+        &.enabled {
+            background-color: var(--core-color-opacity-green-600);
+
+            &:hover {
+                background-color: var(--core-color-opacity-green-800);
+            }
+
+            &:active {
+                background-color: var(--core-color-solid-green-900);
+            }
+        }
+
+        .toggle-switch__knob {
```

```suggestion
        &__knob {
```
and move it outside of `toggle-switch__slider`